csharp/test/Apache.Arrow.Tests/ArrowArrayBuilderFactoryReflector.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Reflection;
+using Apache.Arrow.Types;
+
+namespace Apache.Arrow.Tests
+{
+    static class ArrayArrayBuilderFactoryReflector
+    {
+        private static readonly Type s_arrowArrayBuilderFactoryType = Assembly.Load("Apache.Arrow").GetType("Apache.Arrow.ArrowArrayBuilderFactory");
```

Comment:
Do we need to hold onto this type statically in a field? It could just be inlined into the MethodInfo call below and the field could be removed.

Reply:
It was left for just in case we should call other methods in future, but it was meaningless at least now, so I have removed this type field.